flocker/acceptance/test_environment_variables.py

```diff
@@ -254,3 +254,5 @@ def verify_data_moves(connection_2):
 
         verifying_data_moves = getting_mysql_2.addCallback(verify_data_moves)
         return verifying_data_moves
+
+    test_moving_data.timeout = 10*60
```

Comment:
The tests all passed with your new docker package. http://build.clusterhq.com/builders/flocker%2Facceptance%2Fvagrant%2Ffedora-20/builds/396 so perhaps this isn't strictly necessary here.